lib/screens/categories/add_category_controller.dart

```diff
@@ -98,7 +98,9 @@ class AddCategoryController extends GetxController {
       await dbController.db.update(
           Const.categories, preFilledCategory!.toJson(),
           where: "id = ?", whereArgs: [preFilledCategory!.id]);
-      dbController.categories[preFilledCategory!.id!] = preFilledCategory!;
+      final categoryList = await dbController.db
+          .query(Const.categories, where: 'is_deleted = ?', whereArgs: [0]);
+      dbController.categories.value = categoryFromJson(categoryList);
```

Comment:
what is this? if you set like this, then if transaction list has deleted category, it will mess up.